src/containers/RequestBuilder.js

```diff
@@ -169,197 +175,193 @@ export default class RequestBuilder extends Component {
             console.log(fhirResponse);
             if (fhirResponse?.status) {
-              this.consoleLog(
+              consoleLog(
                 'Server returned status ' + fhirResponse.status + ': ' + fhirResponse.error,
                 types.error
               );
-              this.consoleLog(fhirResponse.message, types.error);
+              consoleLog(fhirResponse.message, types.error);
             } else {
-              this.setState({ response: fhirResponse });
+              setState(prevState => ({ ...prevState, response: fhirResponse }));
             }
-            this.setState({ loading: false });
+            setState(prevState => ({ ...prevState, loading: false }));
           });
         })
         .catch(() => {
-          this.consoleLog('No response received from the server', types.error);
-          this.setState({ response: {} });
-          this.setState({ loading: false });
+          consoleLog('No response received from the server', types.error);
+          setState(prevState => ({ ...prevState, response: {}, loading: false }));
         });
     } catch (error) {
-      this.setState({ loading: false });
-      this.consoleLog('Unexpected error occurred', types.error);
+      
+      setState(prevState => ({ ...prevState, loading: false }));
+      consoleLog('Unexpected error occurred', types.error);
       if (error instanceof TypeError) {
-        this.consoleLog(error.name + ': ' + error.message, types.error);
+        consoleLog(error.name + ': ' + error.message, types.error);
       }
-      this.setState({ loading: false });
     }
-  }
+  };
 
-  takeSuggestion(resource) {
+  const takeSuggestion = (resource) => {
     // when a suggestion is taken, call into the requestBox to resubmit the CRD request with the new request
-    this.requestBox.current.replaceRequestAndSubmit(resource);
-  }
+    requestBox.current.replaceRequestAndSubmit(resource);
+  };
 
-  getPatients = () => {
-    if (this.props.globalState.patientFhirQuery) {
-      this.props.client
-        .request(this.props.globalState.patientFhirQuery, { flat: true })
+  const getPatients = () => {
+    if (globalState.patientFhirQuery) {
+      client
+        .request(globalState.patientFhirQuery, { flat: true })
         .then(result => {
-          this.setState({
-            patientList: result
-          });
+          setState(prevState => ({ ...prevState, patientList: result }));
         })
         .catch(e => {
-          this.setState({
-            patientList: e
-          });
+          setState(prevState => ({ ...prevState, patientList: e }));
         });
     }
   };
 
-  updateStateList = (elementName, text) => {
-    this.setState(prevState => {
-      return { [elementName]: [...prevState[elementName], text] };
-    });
+  const updateStateList = (elementName, text) => {
+    setState(prevState => ({ ...prevState, [elementName]: [...prevState[elementName], text] }));
```

Comment:
```suggestion
```
I don't think this is used anymore (I missed it in my last PR), and this line in PatientSearchBar.js could also be deleted
```js
                callbackList={props.callbackList}
```